CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Version 1.8.2 (Under development)
 
 * **[Fix]** Fix `double-quoted` warnings in Xcode 12.
+* Add an api to customize data path.
```

Comment:
It should be something like this:
`**[Feature]** Add an API to customize data path.`

Also, could you please remove [this](https://github.com/microsoft/plcrashreporter/pull/164/files#r559737820) comment if don't need it.